lib/order.rb

```diff
@@ -0,0 +1,72 @@
+require 'csv'
+require './lib/customer'
+
+class Order
+  STATUSES = [:pending, :paid, :processing, :shipped, :complete]
+  
+  attr_reader :id, :products, :customer, :fulfillment_status
+  
+  def initialize(id, products, new_customer, status = :pending)
+    @id = id
+    @products = products
+    @customer = new_customer
+    raise ArgumentError.new("No fulfillment status were given!") if !STATUSES.include? status
```

Comment:
Your error was misleading.  Also, it's helpful to include the bad status.

```suggestion
    raise ArgumentError.new("Bad fulfillment status given: #{status}!") if !STATUSES.include? status
```